enhancements/machine-config/manage-boot-images.md

```diff
@@ -0,0 +1,206 @@
+---
+title: manage-boot-images
+authors:
+  - "@djoshy"
+reviewers: 
+  - "@yuqi-zhang"
+  - "@mrunal"
+  - "@cgwalters, for rhcos context" 
+  - "@joelspeed, for machine-api context" 
+  - "@sdodson, for installer context"
+approvers:
+  - "@yuqi-zhang"
+api-approvers: 
+  - None
+creation-date: 2023-10-16
+last-updated: 2022-10-17
+tracking-link:
+  - https://issues.redhat.com/browse/MCO-589
+see-also:
+replaces: 
+  - https://github.com/openshift/enhancements/pull/368
+superseded-by: 
+  - https://github.com/openshift/enhancements/pull/201
+---
+
+# Managing boot images via the MCO
+
+## Summary
+
+This is a proposal to manage bootimages via the `Machine Config Operator`(MCO), leveraging some of the [pre-work](https://github.com/openshift/installer/pull/4760) done as a result of the discussion in [#201](https://github.com/openshift/enhancements/pull/201). This feature will only target standalone OCP installs. It will also be user opt-in and is planned to be released behind a feature gate.
+
+For Installer Provisioned Infrastructure(IPI) clusters, the end goal is to create a mechanism that can:
+- update the boot images references in `MachineSets` to the latest in the payload image
+- ensure stub ignition referenced in each `Machinesets` is in spec 3 format
```

Comment:
Do you think it would make sense to also call out the MCS/ignition TLS cert here as a requirement? For some old certs, new bootimages may not be compatible. Doesn't have to make it to the first stage of implementation but we'll have to do it eventually.

Reply:
Ack, will call it out and point to the relevant epic. We can include it one of the later phases